src/controllers/contacts.ts

```diff
@@ -66,10 +68,10 @@ export const getContacts = async (req, res) => {
   })
 
   const subsResponse = subscriptions.map((s) =>
-    jsonUtils.subscriptionToJson(s, null)
+    jsonUtils.subscriptionToJson(s)
   )
   const chatsResponse = chats.map((chat) => {
-    const theChat = chat.dataValues || chat
+    const theChat = chat.dataValues as Chat || chat
```

Comment:
Not sure if the change you just made would apply here aswell, seems another test is failing but it isn't showing an error in the logs

Reply:
Adding types only affects compilation, not tests as when compiling types are removed. Types are a nice way to speed up development and debugging, it doesnt change the resulting js files from typescript compiler

Reply:
Adding this makes ts certain `theChat` is a `Chat` and gives better completions in an ide

Reply:
yea that makes sense, I'm just trying to track down what change made the test start failing